include/os.h

```diff
@@ -294,12 +294,6 @@ _X_ATTRIBUTE_PRINTF(1, 2);
 extern _X_EXPORT void
 xorg_backtrace(void);
 
-#include <signal.h>
```

Comment:
Shouldn’t we keep it (for non win32)?

Reply:
IIRC those sites needing it already including it on their own.